qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -438,6 +492,9 @@ def _rampvoltage(self, chan, fg, setvoltage, ramptime):
         self.write(funmssg)
         self.parameters['ch{:02}_v'.format(chan)]._save_val(setvoltage)
 
```

Comment:
We've agreed that this is not an issue, due to syncs being overriden by subsequent calls to `_ramp_voltage`.